src/charm.py

```diff
@@ -521,6 +520,7 @@ def _on_pebble_ready(self, event: PebbleReadyEvent) -> None:
             self._container.make_dir(path=str(self._log_dir), make_parents=True)
             logger.info(f"Created directory {self._log_dir}")
 
+        self._push_default_files()
```

Comment:
I did not add any checks to check if the default schemas and mappers already exist or need to be updated (these files are rarely changed) in the container's file system. I guess it would be OK to simply push them in the `pebble_ready` event. Let me know if it is necessary to add the check.

Reply:
As long as it does not raise an error if the files already exist, I am fine with this.